completion_aggregator/serializers.py

```diff
@@ -164,10 +164,20 @@ def possible(self):
     def percent(self):
         """
         Report the percentage of possible completions earned.
+        """
+        return self.course.percent
+
+    @property
+    def ratio(self):
+        """
+        Report of ratio of possible completions earned.
 
         Returns a float in the range [0.0, 1.0], where 1.0 == 100%.
         """
-        return self.course.percent
+        if self.course.percent:
+            return self.course.percent / 100
+        else:
+            return 0.0
```

Comment:
One last thing: Despite the name, "percent" is already returning a number in [0.0, 1.0].  The same value needs to be returned as "ratio" here.  I know it's confusing, but that was the outcoming of a long bikeshedding conversation with edX.  The platform uses the percent terminology in a number of places, so we changed it, but the v0 api needs to return the same value as "ratio."

Reply:
Is there a test that shows the values returned for this field?

Reply:
@jcdyer there is no test for this, I will add tests.